_posts/2021-09-20-newpipe-0.21.10-released.md

```diff
@@ -0,0 +1,112 @@
+
+- [@Stypox](https://github.com/Stypox) replaced the outdated Android-Universal-Image-Loader library (which has not had any new release in the last 6 years) with the more stable and updated Picasso, a library to download and cache images on Android (think all the thumbnails you see in the app). This fixes a nasty issue where the wrong notification thumbnail size could crash the whole System UI or even forcibly reboot the device! It also makes loading cached images instantaneous, and loading non-cached images smoother.
+
+- [@sherlockbeard](https://github.com/sherlockbeard) changed the 'video not available' image from a static PNG to a vector XML image, allowing it to scale properly for screens of all sizes.
+
+- [@Stypox](https://github.com/Stypox) added the mime-type field to all calls to the SAF file picker, which causes files with the incorrect file format to be greyed out and become non-selectable. This will ensure the user can only choose a ZIP file when importing their database, for example. This should be a great quality-of-life improvement for many users!
+
+### Fixed
+
+- [@litetex](https://github.com/litetex) together with [@Stypox](https://github.com/Stypox) improved the comments parsing for timestamps, which could have resulted in a crash if it happened in the wrong place at the wrong time
+
+- [@Aga-C](https://github.com/Aga-C) added line wrapping to the settings texts, so they don't get truncated anymore.
+
+- [@acti0](https://github.com/acti0) fixed a regression from a past release where the name of the video wasn't being shared when sharing the video URL.
+
+- [@litetex](https://github.com/litetex) fixed a bug where you had to tap the Replay button twice for it to actually work.
+
+- While [@litetex](https://github.com/litetex) fixed the Replay button, they also fixed the direction buttons on DPads for Android TV users: the seekbar now jumps only 10 seconds, instead of several minutes.
+
+- [@litetex](https://github.com/litetex), being on a roll, reworked the error panel shown when an error happens. This results in a clearer error display.
+
+- [@Redirion](https://github.com/Redirion) implemented proper closure of audio streams when playback is stopped. This fixed a longstanding bug where NewPipe would create more and more audio sessions but use only one. This messed with equaliser apps.
+
+- [@TobiGr](https://github.com/tobigr) added proper error handling which might occur when trying to play a video in popup mode or get a search suggestion.
+
+- [@TobiGr](https://github.com/tobigr) also took care of a small UI bug: when one played something in the miniplayer and changed the visibility of the description tab in the meantime, the visibility of the comments tab was changed instead. The app isn't confused anymore, we cannot say the same about the dev...
```

Comment:
Is "miniplayer" here referring to the popup player?

Reply:
```suggestion
- [@TobiGr](https://github.com/tobigr) also took care of a small UI bug: when one played something in the minimized player and changed the visibility of the description tab in the meantime, the visibility of the comments tab was changed instead. The app isn't confused anymore, we cannot say the same about the dev...
```

minimized

Reply:
Thanks, that sounds clearer to me.